dataplane/apigen/ccgen/servergen.go

```diff
@@ -179,6 +179,26 @@ class {{ .ServiceName }} final : public lemming::dataplane::sai::{{ .ServiceName
 {{ range .Funcs }}
 {{- if and .ProtoRPCName (not .IsStreaming) }}
 grpc::Status {{ $.ServiceName }}::{{ .ProtoRPCName }}(grpc::ServerContext* context, const lemming::dataplane::sai::{{ .ProtoRequestType }}* req, lemming::dataplane::sai::{{ .ProtoResponseType }}* resp) {
+	LOG(INFO) << "Func: " << __PRETTY_FUNCTION__;
+	{{- if .UseCommonAPI }}
+	{{ if eq .Operation "remove" }}
+	grpc::ClientContext context;
+	{{ if .OidVar -}} auto status = api->{{ .Name }}(req.get_oid()); {{ end }}
+  	{{ if .EntryVar }} 
+	{{ .Args }} entry = {{ .EntryConversionToFunc }}(req); 
+	auto status = api->{{.Name}}(entry);
+	{{ end }}
+	if(!status.ok()) {
```

Comment:
status is enum, the check should be if (status != SAI_STATUS_SUCCESS)

Reply:
Fixed